### ISSUE TITLE
Precondition not reporting ( the battery heater fixed with tesla fw)

### ISSUE THREAD

**Issue by AikenPandora** [bassmaster187/TeslaLogger#1429]
**Describe the bug**
Right now battery heater has always no value attached to itself. It seems it is not reporting (according to log)

Preconditioning always reports a string, but only false, active preconditioning isn‘t logged. 

**To Reproduce**
Charge at DC, Navigate to a Supercharger, charge AC with cold battery or use precondition with cold battery (should heat battery and precondition)

Use Precondition. 

**Expected behavior**
Show Precondition and battery heating same as Tesla-App

**Teslalogger Type**
Raspberry Pi4 / Docker / Selfmade
Did you update the Image with apt-get update & upgrade?

**Comment by Adminius** [comment]
https://github.com/teslamotors/fleet-telemetry/issues/317

**Comment by AikenPandora** [comment]
Battery heater fixed for cars with 2025.8.3 (or newer) and since the pull request https://github.com/bassmaster187/TeslaLogger/pull/1506.

Still a problem with preconditioning.

**Comment by Adminius** [comment]
Do you have this entries in your log if precondition the car?
**DefrostForPreconditioning**

![Image](https://github.com/user-attachments/assets/47af611e-9730-4479-bdc4-2654ed92bba0)

**Comment by AikenPandora** [comment]
No, should I have this always when the climate is on, or just if I use defrost or while charging?

**Comment by Adminius** [comment]
I have this each time when I start precondition via tesla app

**Comment by AikenPandora** [comment]
I had it never, not even during startup of the car, where it at least triggered the battery heater as invalid before 2025.8.3.

**Comment by AikenPandora** [comment]
Since switching to FleetAPI, the Preconditioning hasn‘t been working. My last entries in the database are via OwnerAPI.

**Comment by AikenPandora** [comment]
It seems the fix https://github.com/bassmaster187/TeslaLogger/pull/1461/commits/0161b71353d75ace3e83b6cdd6f68e30fadbdbd6 hasn‘t been applied to my version with docker. Even though I made a lot of manual updates since then. 

Am I making a mistake?

**Comment by Adminius** [comment]
the fix is applied, but the car config maybe not... @bassmaster187

**Comment by AikenPandora** [comment]
Both fixed on 1.63.4
